app/Http/Controllers/CompetitionsController.php

```diff
@@ -37,18 +37,6 @@ public function __construct(UserRepositoryContract $repository, Manager $manager
     }
 
     /**
-     * Display a listing of the resource.
-     *
-     * @return \Illuminate\Http\Response
-     */
-    public function index()
```

Comment:
yea I'm not sure yet that we won't ever need this. I mean I guess we could just put it back in later but doesn't seem to hurt to keep it for now.